README.md

```diff
@@ -138,6 +138,99 @@ class ExampleInstrumentedTest {
 }
 ```
 
+#### Lazy lists testing
+
+:warning: This API is experimental and might change in future!
+
+To test lazy lists such as `LazyRow` or `LazyColumn` you should add `KLazyListNode` into your `ComposeScreen`:
+```Kotlin
+val list = KLazyListNode(
+    semanticsProvider = semanticsProvider,
+    viewBuilderAction = { hasTestTag("LazyList") },
+    itemTypeBuilder = {
+        itemType(::LazyListItemNode)
+        itemType(::LazyListHeaderNode)
+    },
+    positionMatcher = { position -> SemanticsMatcher.expectValue(LazyListItemPosition, position) }
+)
+```
+
+Inside `itemTypeBuilder` function you should register `KLazyListItemNode` types to differentiate elements in lazy list:
+```kotlin
+class LazyListItemNode(
+    semanticsNode: SemanticsNode,
+    semanticsProvider: SemanticsNodeInteractionsProvider,
+) : KLazyListItemNode(semanticsNode, semanticsProvider) {
+    val title: KNode = child {
+        hasTestTag("LazyListItemTitle")
+    }
+}
+
+class LazyListHeaderNode(
+    semanticsNode: SemanticsNode,
+    semanticsProvider: SemanticsNodeInteractionsProvider,
+) : KLazyListItemNode(semanticsNode, semanticsProvider) {
+    val title: KNode = child {
+        hasTestTag("LazyListHeaderTitle")
+    }
+}
+```
+
+Due to lazy list construction element's position might change during the scroll, so we should provide `positionMatcher` to correctly determine element position. This can be achieved in different ways, for example you can determine item position through `TestTag`:
```

Comment:
The element position might be changed during the scroll due to lazy list construction, that’s why we should provide `positionMatcher` to determine the element position correctly. It could be achieved in different ways, for example you can determine item position through `TestTag`